homeassistant/components/shelly/entity.py

```diff
@@ -137,10 +136,11 @@ async def async_restore_block_attribute_entities(
             continue
 
         attribute = entry.unique_id.split("-")[-1]
-        description = BlockAttributeDescription(
+        description = BlockEntityDescription(
```

Comment:
I think this needs to be moved out to the correct platform (sensor vs binary_sensor) somehow

Reply:
Done in 3d6e210